content/03.methods.md

```diff
@@ -19,4 +19,13 @@ talk about the discriminator model and how it works
 talk about calibrating deep learning models with temperature smoothing
 
 ## Experimental Design
-talk about sampling experiment
+Being able to re-use label functions would significantly speed up the annotation process.
+To test the viability of re-use we implemented a sampling with replacement approach.
```

Comment:
```suggestion
We first established a baseline by training a generative model using only distant supervision label functions for the target edge type.
As an example, for the gene-interacts-gene edge type we used label functions that returned a `1` if the pair of genes was included in (LIST THE DATABASES HERE).
We then compared models that also included text and domain-heuristic label functions to these.
We sampled with replacement from these text- and domain-heuristic label functions separately within edge types, across edge types, and from a pool of all label functions.
We compared within-edge-type performance to across-edge-type and all-edge-type performance.
We performed each sampling for XYZ evenly-spaced numbers of label functions for each comparison from one to the total number of possible label functions.
We repeated the sampling process 50 times for each point.
We evaluated both generative and discriminative models at each point, and we recorded the performance of each in terms of the area under the receiver operating characteristic curve (AUROC) and the area under the precision-recall curve (AUPR).
```

Reply:
I think you should make these changes and then go back to revise a bit more. I added AUPR since you had been presenting results for both. Did those go away?

Reply:
These did not go away. I incorporated them into the results section.